tasks/sep/030-revert-mt-cannon/README.md

```diff
@@ -0,0 +1,23 @@
+# ProxyAdminOwner - Set Dispute Game Implementation
```

Comment:
I think we should add a link to the version of the contracts used (seems to be the `v1.8.0-rc.4` solidity file `NestedSignFromJson.s.sol`.

Reply:
The contracts used actually come from multiple different releases because our release process doesn't release all contracts together.  The reference to v1.8.0-rc.4 in the superchain registry is a bit misleading but that's how the registry works - it actually defines the expected versions for all contracts at that snapshot even if the contract version is actually from an older release.  I don't think we want to try and explain that level of complexity here - the important thing is actually what VALIDATION.md already covers which is identifying what is actually being changed between the new and old implementation contracts (and the dispute game contracts are not changing - just the MIPS version).